### PR TITLE
added test from task_11(Verify Opinions on phone page) and relevant pages

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/HomePage.java [hunk 1]
```diff
@@ -101,4 +104,14 @@ public PhoneFinderPage openPhoneFinderPage() {
...
+
+    public BrandModelsPage getBrandPageFromPhoneFinderBox(String brand){
+        for (ExtendedWebElement brandFromBox : listOfBrandsOfPhoneFinderBox) {
+            if (brandFromBox.getText().equalsIgnoreCase(brand)) {
+                brandFromBox.click();
+                return new BrandModelsPage(driver);
+            }
+        }
+        return null;
+    }
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/HomePage.java]: its bad
change

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        listOfUnRateCommentBtn.get(index).click();
+    }
+
+    public void selectSortNewestFirst() {
+        newestFirst.click();
+    }
+
+    public List<Date> getListOfCommentsDate() {
+        List<Date> listOfDate = new ArrayList<Date>();
+        for (ExtendedWebElement score : listOfCommentsDate) {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java]: rename score

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            String stringDate = score.getText();
+            SimpleDateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy");
+            try {
+                Date date = dateFormat.parse(stringDate);
+                listOfDate.add(date);
+            }catch (Exception e) {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java]: bad practice catch Exception
and add space after }

Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java (hunk 2 above)
Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/UserOpinionsPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/HomePage.java (hunk 1 above)
Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/HomePage.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**